pulp/tests/test_pulp.py

```diff
@@ -931,6 +931,44 @@ def test_importMPS_binary(self):
         print("\t Testing reading MPS files - binary variable, no constraint names")
         self.assertDictEqual(_dict1, _dict2)
 
+    def test_unset_objective_value__is_valid(self):
+        """Given a valid problem that does not have
+        and driver in the objective function to direct it's minimisation
```

Comment:
I suppose I mean a coefficient which causes it to converge? Here there is a co-efficient of 0 which provides no incentive, but if it was -1 then it would "drive" x towards 1, for example. I agree it is badly written though, I tried to simplify but can also remove entirely and let it speak for itself, let me know.